lib/statsd/instrument/prometheus/flush_stats.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module StatsD
+  module Instrument
+    module Prometheus
+      class FlushStats
+        def initialize(datagrams, default_tags)
+          @datagrams = datagrams
+          @default_tags = default_tags
+        end
+
+        def run
+          datagrams + [flush_stats]
+        end
+
+        private
+
+        attr_reader :datagrams, :default_tags
+
+        def flush_stats
+          DogStatsDDatagram.new(
+            DogStatsDDatagramBuilder.new(default_tags: default_tags).g(
+              "metrics_since_last_flush",
```

Comment:
So we can track how many metrics our processes send over time

Reply:
looks good for now. Wonder what other things we'll want to have insight into later. (Should we create a follow-up ticket to figure if there are things we'll need to know).